packages/xstate-wallet/src/store/memory-channel-storage.ts

```diff
@@ -11,11 +11,13 @@ import {splitSignature, joinSignature} from 'ethers/utils';
 export interface ChannelStorage {
   readonly channelId: string;
   readonly myIndex: number;
-  readonly latestSupported: State | undefined;
-  readonly latestSupportedByMe: State | undefined;
+  readonly latest: StateVariables | undefined; // TODO: Should this always be defined?
```

Comment:
I'm not sure if you'd ever call `createChannel` without passing the first state. It might be worth it to _require_ the first state, and in that case it should always be defined.